app/views/catalog/_in_process_notification_default.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if @document.wayfinder.try(:in_process_file_sets_count).to_i > 0 %>
+  <div class="alert alert-warning" role="alert">Files Processed: <%= @document.wayfinder.processed_file_sets_count %> / <%= @document.wayfinder.processed_file_sets_count + @document.wayfinder.in_process_file_sets_count %></div>
+<% end %>
+<% if @document.resource.try(:pending_uploads).present? %>
+  <div class="alert alert-warning" role="alert">Files currently attaching: <%= @document.resource.pending_uploads.length %></div>
```

Comment:
We should align this heading with the one on the file manager page. I think users will wonder what is the difference between a file attaching and a pending upload.